packages/main/test/pages/DialogAndOpenUI5Dialog.html

```diff
@@ -21,13 +21,14 @@
 			data-sap-ui-compatVersion="edge"></script>
```

Comment:
we need more samples
- openui5 dialog with several ui5-webcomponents inputs like select, combobox and input
- webc dialog with several openui5 inputs
- openui5 popover with webc inputs
- webc popover with openui5 inputs